apps/web/components/DestinationCalendarSelector.tsx

```diff
@@ -11,13 +12,15 @@ interface Props {
   hidePlaceholder?: boolean;
   /** The external Id of the connected calendar */
   value: string | undefined;
+  destinationCalendar?: DestinationCalendar | null;
```

Comment:
DestinationCalendar its now being received as prop to show it as placeholder when no option is selected.